### ISSUE TITLE
TypeScript 2.4.1 not working in VS 2015 Update 3 correctly

### ISSUE THREAD

**Issue by maerlynflagg** [Microsoft/TypeScript#17136]
**TypeScript Version:**  2.4.1
** Visual Studio:** 
Microsoft Visual Studio Enterprise 2015
Version 14.0.25431.01 Update 3
Microsoft .NET Framework
Version 4.6.01586

**Example of the "issue" in a ts file**
![typescript_error](https://user-images.githubusercontent.com/13437496/28134128-53571a1c-6742-11e7-9209-3e35cbfcefd3.png)

** what i have done **
 
I downloaded and install TypeScript 2.4.1 (upgrade from 2.3.1 to 2.4.1):
https://www.microsoft.com/en-us/download/details.aspx?id=48593

I installed over NPM:
Microsoft.TypeScript.Compiler ~2.4.1
Microsoft.TypeScript.MSBuild ~2.4.1

**Expected behavior:**

working ts files (building, autocomplete, intellisense, right colours)

**Actual behavior:**

the font-color of source code is completly white  (in Visual Studio Dark Theme). comments are not green, key words like "interface", "export" not blue. autocomplete etc not working. it looks like v2.4.1 not working with VS2015. i can't build.

code in d.ts files are white text too. 

it feels like programming with a simple text editor. :D

**Downgrade to 2.3.3 issue like version 2.4.1, but i can build!**

**Downgrade to 2.3.1**
before the update to 2.4.1, ts files working fine in visual studio 2015. now i downgrade from 2.4.1. to the version before update (2.3.1) and is also broken (white text etc). i detected another error, which came after the update to 2.4.1, when i try to open a js file in VS. 
 this error is after downgrades too.

![typescript_error2](https://user-images.githubusercontent.com/13437496/28136360-b58d5f14-6749-11e7-86d1-f40d6c8f836b.jpg)

**Comment by maerlynflagg** [comment]
i found solution:

- close VS
- deinstall all typescript versions
- open cmd admin
- go to  cd C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\IDE
- devenv /updateconfiguration
- devenv /clearcache
- install TypeScript 2.4.1
- Open Solution

now all is fine...

**Comment by jpalala** [comment]
I'm using Visual Studio 2017 ("Program Files (x86)\Visual Studio 10.0" in my case), the `devenv` output says that I should do  `/updateconfiguration` and `/clearcache` in project directory. 

So Instead of navigating to `C:\Program Files (x86)\<Microsoft Visual Studio Dir>\Common7\IDE` I navigated to my project root (where the solution files are) and ran the two commands ( `devenv /updateconfiguration` & `devenv /clearcache`) there.

Basically I'm trying to get my version of typescript to an older version which is compatible with the project I'm working on and just adding this comment for reference

**Comment by mhegazy** [comment]
> Basically I'm trying to get my version of typescript to an older version which is compatible with the project I'm working on and just adding this comment for reference

Are the [steps in the update doc](https://github.com/Microsoft/TypeScript/wiki/Updating-TypeScript-in-Visual-Studio-2017#setting-typescript-versions-in-visual-studio-2017-version-153) not working for you?
